sympy/geometry/ellipse.py

```diff
@@ -835,23 +835,23 @@ def normal_lines(self, p, prec=None):
         >>> e = Ellipse((0, 0), 2, 3)
         >>> c = e.center
         >>> e.normal_lines(c + Point(1, 0))
-        [Line(Point2D(0, 0), Point2D(1, 0))]
+        [Line2D(Point2D(0, 0), Point2D(1, 0))]
         >>> e.normal_lines(c)
-        [Line(Point2D(0, 0), Point2D(0, 1)), Line(Point2D(0, 0), Point2D(1, 0))]
+        [Line2D(Point2D(0, 0), Point2D(0, 1)), Line2D(Point2D(0, 0), Point2D(1, 0))]
 
         Off-axis points require the solution of a quartic equation. This
         often leads to very large expressions that may be of little practical
         use. An approximate solution of `prec` digits can be obtained by
         passing in the desired value:
 
         >>> e.normal_lines((3, 3), prec=2)
-        [Line(Point2D(-38/47, -85/31), Point2D(9/47, -21/17)),
-        Line(Point2D(19/13, -43/21), Point2D(32/13, -8/3))]
+        [Line2D(Point2D(-0.81, -2.7), Point2D(9/47, -21/17)),
```

Comment:
Since the evaluated points are Points when the Line is created their decimal approximations are not recomputed to fractions. I think this can be fixed by wrapping the numerically approximated point in `Point`: `points = [Point(pt.n(prec)) for pt in points]` at line 895.